src/documentation/pages/components/forms/search-input/examples/_sizes.vue

```diff
@@ -0,0 +1,160 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes" padding="-p--0" :tabs="exampleTabs">
```

Comment:
this file should be named _sizes-lumen-centurylink and the component example's id should reflect this too, i.e. id="sizes-lumen-centurylink"